aiida/tools/importexport/common/archive.py

```diff
@@ -130,7 +268,7 @@ def meta_data(self):
         :return: dictionary with contents of meta data file
         """
         if self._meta_data is None:
-            self._meta_data = self._read_json_file(self.FILENAME_METADATA)
+            self._meta_data = self._read_json_file(FILENAME_METADATA)
```

Comment:
Why did you make `FILENAME_METADATA` and `FILENAME_DATA` global variables. Do they need to be accessed outside of this class? Wouldn't it make more sense to make these class attributes

Reply:
These are used in the `extract_*` functions - outside the class.